packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -118,6 +119,7 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> {
     let isRTL = getRTL();
     let isOnRightSide = isRTL ? isLeft : !isLeft;
     const headerTextId = id + '-headerText';
+    const customWidthStyles = (type === PanelType.custom) ? { width: customWidth } : {};
```

Comment:
Without media queries, it seems custom widths would be troublesome. We might be able to get away with setting that width, and then a max-width of 80% or something.